src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -854,7 +854,7 @@ def endpoints(cls):
                 'permission': None,
             }
 
-            if func.__name__ in cls._method_mapping:
+            if func.__name__ in cls._method_mapping or hasattr(func, "_method_map_method_"):
```

Comment:
This will work I think. You are checking the other condition below, right?
```suggestion
            if func.__name__ in cls._method_mapping:

Reply:
Done.